recipes/mpir/all/conanfile.py

```diff
@@ -37,7 +37,9 @@ def _vcxproj_path(self):
                                                    "{}_mpir_gc.vcxproj".format(self._dll_or_lib))
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version], keep_permissions=True)
+        args = self.conan_data["sources"][self.version]
+        args["keep_permissions"] = True
+        tools.get(**args)
```

Comment:
```suggestion
        tools.get(keep_permissions=True, **self.conan_data["sources"][self.version])
```

This should be valid in Python2